galsim/real.py

```diff
@@ -660,3 +640,30 @@ def simReal(real_galaxy, target_PSF, target_pixel_scale, g1=0.0, g2=0.0, rotatio
 
     # return simulated image
     return image
+
+def parse_files_dirs(file_name, image_dir, dir, noise_dir):
```

Comment:
This should probably get a leading underscore.  It's an implementation detail, so should be "hidden".